packages/core/src/ceramic.ts

```diff
@@ -614,6 +615,12 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
       // If authenticated into the node, we can start publishing metrics
       // publishing metrics is enabled by default, even if no metrics config
       if (this._metricsConfig?.metricsPublisherEnabled) {
+        // First, subscribe the node to the Model used for NodeMetrics
+        const metricsModel = NodeMetrics.getModel(this._networkOptions.name)
+        await this.repository.index.indexModels([{ streamID: metricsModel }])
```

Comment:
is there a possible failure here?  what if we fail to getModel, and its null, will this throw an error?

Reply:
Yes it will, would prefer to do all error handling cleanup over in https://github.com/ceramicnetwork/js-ceramic/pull/3265